build.gradle

```diff
@@ -41,6 +41,7 @@ task coverage(type: JacocoReport) {
 }
 
 dependencies {
+    implementation 'junit:junit:4.13.1'
     String jUnitVersion = '5.4.0'
```

Comment:
this seems weird, can we stick to junit 5?
Might need to change all the `@Ignore` to `@Disabled`

Reply:
This was what it came as, maybe we can discuss a bit more first?